plugins/source/aws/resources/services/ssoadmin/permission_sets.go

```diff
@@ -35,38 +33,23 @@ The 'request_account_id' and 'request_region' columns are added to show the acco
 				Resolver: client.ResolveAWSRegion,
 			},
 			{
-				Name:     "inline_policy",
-				Type:     sdkTypes.ExtensionTypes.JSON,
-				Resolver: getSsoadminPermissionSetInlinePolicy,
+				Name:       "instance_arn",
+				Type:       arrow.BinaryTypes.String,
+				Resolver:   schema.ParentColumnResolver("instance_arn"),
+				PrimaryKey: true,
 			},
 		},
 
 		Relations: []*schema.Table{
 			accountAssignments(),
+			inlinePolicies(),
+			customerMangedPolices(),
```

Comment:
See comment above on spelling for Managed and Policies.

Reply:
fixed in https://github.com/cloudquery/cloudquery/pull/10985/commits/00c529da2c80c93084346378b9026edd554ee68d